pkg/provider/loadbalancerclass_test.go

```diff
@@ -2,6 +2,8 @@ package provider
 
 import (
 	"context"
+	"k8s.io/api/core/v1"
```

Comment:
This should be gone once https://github.com/kube-vip/kube-vip-cloud-provider/pull/137/files#r1525580223 is fixed